torchtitan/config_manager.py

```diff
@@ -465,6 +465,12 @@ class Float8:
     Not compatible with torch.compile.
     """
 
+    moe_fqns: list[str] | str = field(default_factory=list)
```

Comment:
can we add "prototype" to the field name and add a link to the README in the docstring